extensions/ql-vscode/src/extension.ts

```diff
@@ -344,6 +345,14 @@ export async function activate(
     codeQlExtension.variantAnalysisManager,
   );
 
+  try {
+    await prepareCodeTour();
```

Comment:
Could we move this call to be before activating the extension? It doesn't seem like this uses the extension or CLI, so by moving it up above the `installOrUpdateThenTryActivate` call, we can significantly reduce the lag between the user opening VS Code and reloading the workspace mentioned by @shati-patel.

Reply:
Ah nice! I'll give it a shot, thanks! :)

Reply:
Alrighty, I've moved this up and it looks like the prompt to open the workspace appears immediately. 

I'm unable to test this on windows, but at least we know it still works.

@shati-patel are you willing to try it out one more time?